cli.js

```diff
@@ -5,6 +5,7 @@ var path = require('path');
 var globby = require('globby');
 var meow = require('meow');
 var updateNotifier = require('update-notifier');
+require('babel/register');
```

Comment:
I think we should try to load the local `babel` before using the built-in one. So the user can provide their own.

You can use https://github.com/sindresorhus/resolve-from to achieve this.